tests/unit_test/data_processor/upload_handler_log_processor_tests.py

```
@@ -0,0 +1,122 @@
+        assert log[0]['start_timestamp']
+        assert log[0].get('end_timestamp')
+        assert log[0]['file_name'] ==  "Salesstore.csv"
+        assert log[0].get('status') == 'Success'
+        assert log[0]['event_status'] == EVENT_STATUS.COMPLETED.value
+
+    def test_is_limit_exceeded_exception(self, monkeypatch):
+        bot = 'test_bot'
+        try:
+            bot_settings = BotSettings.objects(bot=bot).get()
+            bot_settings.content_importer_limit_per_day = 0
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)